outputs/policyadapter.go

```diff
@@ -57,4 +63,65 @@ func NewPolicyReportClient(config *types.Configuration, stats *types.Statistics,
 // PolicyReportPost creates Policy Report Resource in Kubernetes
 func (c *Client) PolicyReportCreate(falcopayload types.FalcoPayload) {
 	//to do
+	var crdclient *crdClient.Clientset
+	if c.Config.PolicyReport.Kubeconfig != "" {
+		restConfig, err := clientcmd.BuildConfigFromFlags("", c.Config.PolicyReport.Kubeconfig)
+		if err != nil {
+			fmt.Printf("unable to load kube config file: %v", err)
+		}
+		crdclient, err = crdClient.NewForConfig(restConfig)
+		if err != nil {
+			fmt.Printf("u %v", err)
+		}
+	}
+	restConfig, err := rest.InClusterConfig()
+	if err != nil {
+		fmt.Printf("unable to load in-cluster config: %v", err)
+	}
+
+	crdclient, err = crdClient.NewForConfig(restConfig)
+	if err != nil {
+		fmt.Printf("u %v", err)
+	}
```

Comment:
I don't understand this block, why do you recreate a client and not use `c.KubernetesClient` ?

Reply:
I don't understand this block, why do you recreate a client and not use c.KubernetesClient ?
Hey thomas, I have modified the code to be little more efficient, but i think we will need this as we need a client interacting with the CRD. I am creating that using
```
restConfig, err := rest.InClusterConfig()
	if err != nil {
		restConfig, err = clientcmd.BuildConfigFromFlags("", c.Config.PolicyReport.Kubeconfig)
		if err != nil {
			fmt.Printf("unable to load kube config file: %v", err)
		}
	}
	crdclient, err = crdClient.NewForConfig(restConfig)
	if err != nil {
		fmt.Printf("u %v", err)
	}
ats := crdclient.Wgpolicyk8sV1alpha2().ClusterPolicyReports()
```

Reply:
it makes more sense to create this client in the NewPolicyReportClient, nothing disallows to create 2 "k8s clients" in one object `Client`